LibreNMS/Validations/User.php

```diff
@@ -106,6 +106,11 @@ public function validate(Validator $validator): void
                         "$dir/.pki/", // ignore files/folders created by setting the librenms home directory to the install directory
                     ];
 
+                    // files are restored before update and modified when plugins are installed
+                    if (file_exists("$dir/composer.plugins.json")) {
```

Comment:
This validation is for files not owned by the LibreNMS user. Surely we don't want to skip these files otherwise we can't do an update via daily.sh

Reply:
@laf consider this bit of code:

https://github.com/librenms/librenms/blob/master/daily.sh#L296

Reply:
wait was this the wrong validation? :D